core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -305,4 +334,76 @@ public void start() {
         }
         logger().info("Docker compose has finished running");
     }
-}
\ No newline at end of file
+}
+
+class LocalDockerCompose implements DockerCompose {
+    private final File composeFile;
+    private final String identifier;
+    private String cmd = "";
+    private Map<String, String> env = new HashMap<>();
+
+    public LocalDockerCompose(File composeFile, String identifier) {
+        this.composeFile = composeFile;
+        this.identifier = identifier;
+    }
+
+    @Override
+    public DockerCompose withCommand(String cmd) {
+        this.cmd = cmd;
+        return this;
+    }
+
+    @Override
+    public DockerCompose withEnv(Map<String, String> env) {
+        this.env = env;
+        return this;
+    }
+
+    @Override
+    public void start() {
+        try {
+            final ProcessBuilder pb = new ProcessBuilder(Splitter.onPattern(" ")
```

Comment:
ah - please use [Zeroturnaround Exec](https://github.com/zeroturnaround/zt-exec) (already on the class path!) for this. It will make a lot of things easier, including pumping stdout/stderr to the logger.